lib/Extension/ObjectRenderer/ObjectRendererBuilder.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Phpactor\Extension\ObjectRenderer;
+
+use Closure;
+use Phpactor\ObjectRenderer\Adapter\Twig\TwigObjectRenderer;
+use Phpactor\ObjectRenderer\Model\ObjectRenderer;
+use Phpactor\ObjectRenderer\Model\ObjectRenderer\TolerantObjectRenderer;
+use Phpactor\ObjectRenderer\Model\TemplateCandidateProvider;
+use Phpactor\ObjectRenderer\Model\TemplateProvider\AncestoralClassTemplateProvider;
+use Phpactor\ObjectRenderer\Model\TemplateProvider\ClassNameTemplateProvider;
+use Phpactor\ObjectRenderer\Model\TemplateProvider\InterfaceTemplateProvider;
+use Phpactor\ObjectRenderer\Model\TemplateProvider\SuffixAppendingTemplateProvider;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+/**
+ * Object Renderer builder.
+ *
+ * A new instance of the builder is returned for each modification.
+ */
+final class ObjectRendererBuilder
```

Comment:
copied verbatim from the object-renderer library --- allowing Twig to be configured